base_classes/NXdisk_chopper.nxdl.xml

```diff
@@ -24,13 +24,26 @@
 <definition xmlns="http://definition.nexusformat.org/nxdl/3.1" category="base"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
-	name="NXdisk_chopper" 
+	name="NXdisk_chopper"
 	type="group" extends="NXobject">
-	
+
+	<symbols>
+    <doc>This symbol will be used below to coordinate datasets with the same shape.</doc>
+    <symbol name="n"><doc>Number of slits in the disk</doc></symbol>
+  </symbols>
+
 	<doc>
-		A device blocking the beam in a temporal periodic pattern.
-		
-		TODO: need documentation
+		A disk which blocks the beam but has one or more slits to periodically
```

Comment:
First line of this documentation should be a single-line summary (to be used in creating this summary: http://download.nexusformat.org/doc/html/classes/base_classes/index.html#base-class-definitions).  A blank line afterwards, then any multi-line text.